src/View/Artist/search.html.twig

```diff
@@ -1,25 +1,22 @@
 {% extends 'layout.html.twig' %}
-{% set active_page = "/artists" %}
+{% block title %}Artistes
+{% endblock %}
 
 {% block content %}
-	<main class="container">
-		<div class="row">
-			<h2>Trouve des artistes</h2>
-		</div>
-		{% include 'Artist/_searchArtiste.html.twig' %}
-
-		<div id="allArtists" class="container marketing">
-			<h3>Suggestions</h3>
-			<div class="row">
-				{% for artist in artists %}
-					<div class="col-6 col-lg-4 col-xl-3 text-align-center">
-						<img src="{{ artist.image_artist }}" class="bd-placeholder-img rounded-circle shadow-lg" role="img" preserveaspectratio="xMidYMid slice" focusable="false">
-					</img>
-
-					<p class="artist-name">{{ artist.name_artist }}</p>
+	<div class="row">
+		<h2>Trouve des artistes</h2>
+	</div>
+	{% include 'Artist/_searchArtiste.html.twig' %}
 
-				</div>
-			{% endfor %}
-		</div>
+	<div id="allArtists" class="container marketing">
+		<h3>Suggestions</h3>
+		<div class="row">
+			{% for artist in artists %}
+				<div class="col-6 col-lg-4 col-xl-3 text-align-center">
+					<img src="{{ artist.image_artist }}" class="bd-placeholder-img rounded-circle shadow-lg" role="img" preserveaspectratio="xMidYMid slice" focusable="false">
+				</img>
```

Comment:
`<img>` tags doesn't close : https://www.w3schools.com/tags/tag_img.asp

Reply:
delete search page not used